Xamarin.Forms.Platform.WPF/Extensions/FormattedStringExtensions.cs

```diff
@@ -17,10 +17,13 @@ public static IEnumerable<Inline> ToInlines(this FormattedString formattedString
 
 		public static Run ToRun(this Span span)
 		{
-			var run = new Run { Text = span.Text };
+			var run = new Run { Text = span.Text ?? string.Empty };
 
 			if (span.TextColor != Color.Default)
 				run.Foreground = span.TextColor.ToBrush();
+			
+			if (span.BackgroundColor != Color.Default)
+				run.Background = span.BackgroundColor.ToBrush();
```

Comment:
While UWP doesn't support a background color in a Run, WPF does, hence I just added it here. A quick sneaky win, while WPF is still in preview :)

Reply:
Very nice!